### PR TITLE
Docker Image for KPSR Tutorial

### PR DESCRIPTION
**Pull request by caiotoledo**

### CODE REVIEW

**Review by pablogh-2000 — COMMENTED** [review]
If ready, please elevate to PR

File: .github/workflows/pushdockerimage.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  run:
+    shell: bash
+
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  docker:
+    strategy:
```
Comment (tomasz3dk) [inline .github/workflows/pushdockerimage.yml:33]: Strategy is not required. We don't have matrix here.

File: .github/workflows/pushdockerimage.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          username: ${{ github.actor }}
+          password: ${{ secrets.GH_TOKEN }}
+
+      - name: Build and push KPSR Tutorial Docker image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./ci/docker/kpsrtutorial-1.0.0.dockerfile
+          push: true
+          tags: ${{ env.REGISTRY }}/klepsydra-technologies/kpsrtutorial:latest
```
Comment (tomasz3dk) [inline .github/workflows/pushdockerimage.yml:56]:
```suggestion
          tags: ${{ env.REGISTRY }}/klepsydra-technologies/kpsr-tutorial:latest
```

File: .github/workflows/pushdockerimage.yml (hunk 1 above)
Comment (caiotoledo) [reply .github/workflows/pushdockerimage.yml]: Indeed you're right, I forgot to delete these lines.
Done in 7a4c95b.

File: .github/workflows/pushdockerimage.yml (hunk 2 above)
Comment (caiotoledo) [reply .github/workflows/pushdockerimage.yml]: Name updated in a32e4e3.

**Review by pablogh-2000 — APPROVED** [review]

**Review by tomasz3dk — APPROVED** [review]

**Pull request merged**